client/src/app/project/project.component.ts

```diff
@@ -148,10 +152,10 @@ export class ProjectComponent implements OnInit {
   }
 
   canSubmit(project: Project) {
```

Comment:
Same with the `Account` above: perhaps we should have a `Project.isSubmittable()` method?